next.config.js

```diff
@@ -1,44 +1,54 @@
 const { aliasTinaDev } = require('@tinacms/webpack-helpers')
 
+const MomentLocalesPlugin = require('moment-locales-webpack-plugin')
 const withSvgr = require('next-svgr')
+
 require('dotenv').config()
 
+const withBundleAnalyzer = require('@next/bundle-analyzer')({
+  enabled: process.env.ANALYZE === 'true',
+})
+
 const dummyMailchimpEndpoint =
   'https://theDomainHere.us18.list-manage.com/subscribe/post?u=1512315231251&amp;id=0asd21t12e1'
 
-module.exports = withSvgr({
-  env: {
-    MAILCHIMP_ENDPOINT:
-      process.env.NODE_ENV === 'production'
-        ? process.env.MAILCHIMP_ENDPOINT
-        : dummyMailchimpEndpoint,
-    HUBSPOT_TEAMS_FORM_ID: process.env.HUBSPOT_TEAMS_FORM_ID,
-    HUBSPOT_PORTAL_ID: process.env.HUBSPOT_PORTAL_ID,
-    GTM_ID: process.env.GTM_ID,
-    GITHUB_CLIENT_ID: process.env.GITHUB_CLIENT_ID,
-    BASE_REPO_FULL_NAME: process.env.BASE_REPO_FULL_NAME,
-    BASE_BRANCH: process.env.BASE_BRANCH,
-  },
-  exportTrailingSlash: true,
-  exportPathMap: async function() {
-    return {}
-  },
-  webpack(config) {
-    if (process.env.TINA) {
-      let watch
-      if (process.env.TINA_WATCH) {
-        watch = process.env.TINA_WATCH.split(',')
+module.exports = withBundleAnalyzer(
```

Comment:
Looks good! only thing is we might consider doing something like 
```js
const nextconfig = {
   env{
      //...
   }
//...
}
```
and then
```js
export withBundleAnalyzer(withSvgr(nextconfig));
```
might be cleaner if you add more of these nextjs plugins.

you can look at with[ nextjs.org does here](https://github.com/vercel/next-site/blob/3b270c0644c346af832dcbd5a973a1276f0d4335/next.config.js#L198)

We don't need to make that change now just might make things easier later.

Reply:
Agreed. Might as well do that now.